scripts/launch/demo.sh

```diff
@@ -17,14 +17,19 @@ if [[ ! -d "$base/../../.stack-work" ]]; then
 fi
 
 # Define the default amount of nodes to run
-DEFAULT_NODES_N=3
+DEFAULT_NODES_N=4
 
 # Detect node number or use default
 n=$1
 if [[ "$n" == "" ]]; then
   n=$DEFAULT_NODES_N
 fi
 
+if [[ "$n" != 4 ]]; then
+  echo "Currently we don't support any amount of nodes in demo that's not 4. See CSL-1580"
+  exit
```

Comment:
Revert it please. It shouldn't exit in this case.

Reply:
ok

Reply:
`76874b40f4ff723cc5274fe88a2acd0ba7cb877f`